child_compassion/views/child_compassion_view.xml

```diff
@@ -1,11 +1,73 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <!-- Top menu item -->
-    <menuitem name="Sponsorship" id="menu_sponsorship_root" groups="group_sponsorship" sequence="0"/>
+    <menuitem id="menu_sponsorship_root"
+              name="Sponsorship"
+              groups="group_sponsorship"
+              sequence="0"
+              web_icon="child_compassion,static/description/icon.png"/>
+    <menuitem id="menu_sponsorship_section"
+              name="Sponsorship"
+              parent="menu_sponsorship_root"
+              sequence="1"/>
+    <menuitem id="menu_pool_section"
+              name="Global Childpool"
+              parent="menu_sponsorship_root"
+              sequence="2"/>
+    <menuitem id="menu_field_section"
+              name="Field"
+              parent="menu_sponsorship_root"
+              sequence="3"/>
 
-    <menuitem id="menu_sponsorship_section" parent="menu_sponsorship_root" name="Sponsorship" sequence="1"/>
-    <menuitem id="menu_pool_section" parent="menu_sponsorship_root" name="Global Childpool" sequence="2"/>
-    <menuitem id="menu_field_section" parent="menu_sponsorship_root" name="Field" sequence="3"/>
+    <!--Compassion child kanban view -->
+    <record model="ir.ui.view"
+            id="view_compassion_child_kanban">
+        <field name="name">compassion.child.kanban</field>
+        <field name="model">compassion.child</field>
+        <field name="type">kanban</field>
+        <field name="arch"
+               type="xml">
+            <kanban>
+                <field name="state"/>
+                <field name="portrait"/>
+                <templates>
+                    <t t-debug="pdb"
+                       t-name="kanban-box">
+                        <div class="oe_kanban_vignette">
+                            <!-- Left side image: -->
+                            <div class="oe_kanban_details">
+                                <!-- Title and data -->
+                                <div class="oe_product_desc">
+                                    <img class="oe_kanban_image"
+                                         t-att-src="kanban_image('compassion.child', 'portrait', record.id.raw_value)"/>
+                                    <h4>
+                                        <a type="open">
+                                            <field name="name"></field>

```

Comment:
When you have nothing inside a tag, you can use the shortcut notation `<field name="name"/>`